modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -509,10 +516,17 @@ public boolean isFormAvailable() throws PortalException {
 			if ((group != null) && group.isStagedRemotely()) {
 				ThemeDisplay themeDisplay = getThemeDisplay();
 
+				Role administratorRole = _roleLocalService.getRole(
+					themeDisplay.getCompanyId(), RoleConstants.ADMINISTRATOR);
+
+				List<User> users = _userLocalService.getRoleUsers(
+					administratorRole.getRoleId());
+
+				User administratorUser = users.get(0);
```

Comment:
@rijema or @linolaoj, please avoid create variables only to use as parameters of a method, instead use as parameter `users.get(0)`. According to Brian, usually, variables should be created when they need to be used on 2 or more places in the code. :wink: